element-hq/element-x-android
# Ensure starting PinUnlockActivity does not crash the application.

## Content


Fix regression from #3222 d4f8052696f69521c86f13115a703d854ac325c4

## Motivation and context


Close #3353 

## Screenshots / GIFs



## Tests



- Set up a pin code
- Kill the app 
- Start the app
- the app should not crash now.

## Tested devices

- OS version(s):

## Checklist
